docs/crafted-emacs.org

```diff
@@ -131,11 +131,28 @@ for the code examples.
   install packages:
 
   #+begin_src emacs-lisp
-    (setq crafted-package-installer #'straight-use-package)
-    (setq crafted-package-installed-predicate #'straight--installed-p)
-    (crafted-package-install-package-list)
+  (setq crafted-package-installer #'straight-use-package)
+  (setq crafted-package-installed-predicate #'straight--installed-p)
+  (crafted-package-install-package-list)
   #+end_src
 
+  You can find an example to bootstrap straight in the examples directory:
```

Comment:
I'd personally keep the "You can find an example" part, but remove the copying to custom-modules block.